### PR TITLE
RHDEVDOCS-3389 Document Grafana removal from Monitoring stack

### PR DESCRIPTION
**Pull request by bburt-rh**
This PR removes from the documentation all references to the cluster monitoring Grafana dashboard component, which has been removed from the Monitoring stack and is no longer installed as part of OCP.

NOTE: The Grafana references in the **Observe** > **Accessing third-party UIs** section of the OCP documentation will be removed as part of a different PR: https://github.com/openshift/openshift-docs/pull/38652

- Aligned team: DevTools
- For branches: 
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3389
- Link to doc preview: https://deploy-preview-38687--osdocs.netlify.app/
- SME review: @kyoto  and @simonpasquier 
- QE review: @juzhao 
- Peer review: @ tbd
- <All reviews complete. Please merge now>

### CODE REVIEW

**Comment by lihongyan1** [comment]
> This PR removes from the documentation all references to the cluster monitoring Grafana dashboard component, which has been removed from the Monitoring stack and is no longer installed as part of OCP.
> 
> NOTE: The Grafana references in the **Observe** > **Accessing third-party UIs** section of the OCP documentation will be removed as part of a different PR: #38652
> 
> * Aligned team: DevTools
> * For branches:
> * Jira: https://issues.redhat.com/browse/RHDEVDOCS-3389
> * Link to doc preview: https://deploy-preview-38687--osdocs.netlify.app/
> * SME review: @kyoto  and @simonpasquier
> * QE review: @juzhao
> * Peer review: @ tbd
> *

Grafana link is removed from UI and customer can't access it directly, but still can access it by getting url from network route, for sure grafana is still installed as part of monitoring stack

**Comment by lihongyan1** [comment]
Before merge the PR, we should make sure the 'Grafana removal' epic is completed
https://issues.redhat.com/browse/MON-1591

File: scalability_and_performance/scaling-cluster-monitoring-operator.adoc [hunk 1]
```diff
@@ -5,11 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} exposes metrics that the Cluster Monitoring Operator collects and stores in the Prometheus-based monitoring stack. As an administrator, you can view system resources, containers and components metrics in one dashboard interface, Grafana.
+{product-title} exposes metrics that the Cluster Monitoring Operator collects and stores in the Prometheus-based monitoring stack. As an administrator, you can view system resources, containers and components metrics in the {product-title} web console in *Monitoring* > *Dashboards*.
```
Comment (lihongyan1) [inline scalability_and_performance/scaling-cluster-monitoring-operator.adoc:8]: should be in *Observe* > *Dashboards*.

Comment (juzhao) [reply scalability_and_performance/scaling-cluster-monitoring-operator.adoc:8]: same for https://deploy-preview-38687--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-dashboards.html#cluster-logging-dashboards-access_cluster-logging-dashboards

**Comment by juzhao** [comment]
FYI, https://deploy-preview-38687--osdocs.netlify.app/openshift-enterprise/latest/monitoring/understanding-the-monitoring-stack.html
the monitoring structure diagram has Grafana, it is better to remove it

File: scalability_and_performance/scaling-cluster-monitoring-operator.adoc (hunk 1 above)
Comment (juzhao) [inline scalability_and_performance/scaling-cluster-monitoring-operator.adoc:8]: *Monitoring* > *Dashboards* should be *Observe* > *Dashboards*

File: modules/understanding-quick-start-elements.adoc [hunk 2]
```diff
@@ -147,7 +147,7 @@ image::quick-start-icon.png[quick start icon element in the web console]
 
     - Kubernetes health probes and integration with Spring Actuator
 
-    - Metrics: Prometheus, Grafana, and integration with Spring Cloud Sleuth
+    - Metrics: Prometheus and integration with Spring Cloud Sleuth
```
Comment (juzhao) [inline modules/understanding-quick-start-elements.adoc:150]: no need to remove, not related to cluster monitoring Grafana

**Comment by juzhao** [comment]
https://deploy-preview-38687--osdocs.netlify.app/openshift-enterprise/latest/post_installation_configuration/network-configuration.html
remove `<apps_FIP> grafana-openshift-monitoring.apps.<cluster name>.<base domain>`

**Comment by juzhao** [comment]
For modules/configuring-cluster-monitoring.adoc
I think we should fix https://bugzilla.redhat.com/show_bug.cgi?id=2060699 first

**Comment by bburt-rh** [comment]
@juzhao I will see if we can get https://bugzilla.redhat.com/show_bug.cgi?id=2060699 for the next release

**Pull request closed without merging**